### PR TITLE
Load channel members with channels to make sure we have latest unread counts

### PR DESCRIPTION
**Pull request by jwilander**
#### Summary
This should fix channels being randomly marked unread for messages you've sent/seen.

### CODE REVIEW

**Comment by jwilander** [comment]
Weird, I didn't touch that code. Let me re-run it and see what happens

File: webapp/actions/global_actions.jsx [hunk 1]
```diff
@@ -140,15 +141,15 @@ export function doFocusPost(channelId, postId, data) {
         channelId,
         post_list: data
     });
-    AsyncClient.getChannels(true);
+    loadChannelsForCurrentUser();
```
Comment (crspeller) [inline webapp/actions/global_actions.jsx]: Do we need to do this (or the next line) here? It's just like switchToChannel.

File: webapp/actions/global_actions.jsx [hunk 2]
```diff
@@ -140,15 +141,15 @@ export function doFocusPost(channelId, postId, data) {
...
     AsyncClient.getMoreChannels(true);
     AsyncClient.getChannelStats(channelId);
     loadPostsBefore(postId, 0, Constants.POST_FOCUS_CONTEXT_RADIUS, true);
     loadPostsAfter(postId, 0, Constants.POST_FOCUS_CONTEXT_RADIUS, true);
 }
 
 export function emitPostFocusEvent(postId, onSuccess) {
-    AsyncClient.getChannels(true);
+    loadChannelsForCurrentUser();
```
Comment (crspeller) [inline webapp/actions/global_actions.jsx]: Same here.

File: webapp/actions/global_actions.jsx (hunk 1 above)
Comment (jwilander) [inline webapp/actions/global_actions.jsx]: Probably not, but I didn't want to go removing a bunch of stuff this close to code complete. Ticket [here](https://mattermost.atlassian.net/browse/PLT-4594) to look at this later.

Comment (crspeller) [inline webapp/actions/global_actions.jsx]: Sounds good.

**Comment by crspeller** [comment]
@jwilander You have some conflicts.

**Pull request merged**